### DISCUSSION TITLE
Detect minimum percentage of cocoa

### DISCUSSION THREAD

**Discussion by teolemon** [openfoodfacts/robotoff#1217] [Ideas]
Detect minimum percentage of cocoa in the packaging
"Cacao: 75% minimum dans le chocolat"
"Cacao: 75% minimum"
"75% de cacao minimum"
"75% minimum de cacao"
"Cacao: 35% min(dans le chocolat au lait)."

**Comment by Gaurav7888** [comment]
@teolemon  I am ready with the python script

**Comment by alexgarel** [comment]
@teolemon can you precise if it's a new insight type ? Where does it belong ?
